### PR TITLE
Reproducible ML sample

### PR DESCRIPTION
**Pull request by giograno**
Creating the sample we are using for our Cloud Pods tutorial (https://github.com/localstack/docs/pull/399).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Some of the samples require LocalStack Pro features. Please make sure to properl
 | [Glue for ETL jobs](glue-etl-jobs)                             | Using Glue API to run local ETL jobs                                                               |
 | [Message Queue broker](mq-broker)                              | Using MQ API to run local message queue brokers                                                    |
 | [ELB Load Balancing](elb-load-balancing)                       | Using ELBv2 Application Load Balancers locally, deployed via the Serverless framework              |
+| [Reproducible ML](reproducible-ml)                             | Train, save and evaluate a sklearn machine learning model using AWS Lambda and S3                  |
```
Comment (whummer) [inline README.md:57]: nit: would suggest we use "scikit-learn", which is the slightly more common project name for the library itself
```suggestion
| [Reproducible ML](reproducible-ml)                             | Train, save and evaluate a scikit-learn machine learning model using AWS Lambda and S3                  |
```

Comment (whummer) [inline README.md:57]: nit: would suggest we use "scikit-learn", which is the slightly more common project name for the library itself
```suggestion
| [Reproducible ML](reproducible-ml)                             | Train, save and evaluate a scikit-learn machine learning model using AWS Lambda and S3                  |
```

Comment (whummer) [inline README.md:57]: nit: would suggest we use "scikit-learn", which is the slightly more common project name for the library itself
```suggestion
| [Reproducible ML](reproducible-ml)                             | Train, save and evaluate a scikit-learn machine learning model using AWS Lambda and S3                  |
```

File: reproducible-ml/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# LocalStack Demo: Train, save and evaluate a sklearn machine learning model
```
Comment (whummer) [inline reproducible-ml/README.md:1]:
```suggestion
# LocalStack Demo: Train, save and evaluate a scikit-learn machine learning model
```

File: reproducible-ml/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+## Running
+
+The entire workflow is executed by the `run.sh` script. To trigger it, execute:
+```
+make run
+```
+The model will be first trained by the `ml-train` Lambda function and then uploaded on the S3 bucket.
+A second Lambda function will download the model and evaluate its performance.
```
Comment (whummer) [inline reproducible-ml/README.md:38]:
```suggestion
A second Lambda function will download the model and run predictions on a test set of character inputs.
```

**Review by whummer — APPROVED** [review]
Cool, great to see this sample added @giograno ! 🚀 

Only added a minor comment/suggestion regarding the wording of running the predictions through the model - technically we're not performing model evaluation here (only running prediction for the test set). If we wanted to include model evaluation as well, we could include a classification report as well, see details from the tutorial here: https://scikit-learn.org/stable/auto_examples/classification/plot_digits_classification.html#classification
```
print(
    f"Classification report for classifier {clf}:\n"
    f"{metrics.classification_report(y_test, predicted)}\n"
)
```
